src/Compiler/TypedTree/TypedTreeOps.fs

```diff
@@ -10571,23 +10571,42 @@ let (|EmptyModuleOrNamespaces|_|) (moduleOrNamespaceContents: ModuleOrNamespaceC
             None
     | _ -> None
 
-let tryAddExtensionAttributeIfNotAlreadyPresent
+let tryFindExtensionAttribute (attribs: Attrib list): Attrib option =
+     List.tryFind
+         (fun (a: Attrib) ->
+            a.TyconRef.CompiledRepresentationForNamedType.BasicQualifiedName = "System.Runtime.CompilerServices.ExtensionAttribute")
+         attribs
```

Comment:
I am wondering if this should go via usual mechanism in TcGlobals (where we do `findSysAttrib`/`findOrCreateSysAttrib` and then use TryFindILAttribute/TryFindFSharpAttribute)? The same way we do for `IsReadOnlyAttribute` for example.

Reply:
`TcGlobals` already has this:

`  member val attrib_ExtensionAttribute = findSysAttrib "System.Runtime.CompilerServices.ExtensionAttribute"`

Reply:
Thanks, for pointing this out. I've tried to address this.
Let me know if it is more or less what you had in mind.

Reply:
This is good, thanks for the update.